controllers/endpoints.controller.js

```diff
@@ -0,0 +1,7 @@
+const { fetchEndpoints } = require("../models/endpoints.model");
+
+exports.getEndpoints = (req, res, next) => {
+	fetchEndpoints().then((endpoints) => {
+		res.send({ endpoints });
+	});
```

Comment:
Well named function name: `getEndPoints` and good to see you're sending the response back on a key of `endpoints`, this is appropiate.